h/search/connection.py

```diff
@@ -12,15 +12,25 @@
 # TODO: Make `hosts` a required argument
 def connect(alias='default', hosts=[ELASTICSEARCH_URL], **kwargs):
     """
-    Establish a 'default` connection to Elasticsearch
+    Establish a connection to Elasticsearch through elasticsearch_dsl
 
-    This establishes a connection to newer (v6.x) Elasticsearch which is
-    available as the 'default' connection henceforth via `elasticsearch_dsl`
+    This establishes a connection to newer (v6.x) Elasticsearch. Unless the
+    `alias` parameter has a value other than "default", this connection will
+    then be available as the 'default' connection for `elasticsearch_dsl`
 
-    e.g. `elasticsearch_dsl.connections.get_connection()`
+    e.g. `elasticsearch_dsl.connections.get_connection()` will return the
+    connection established here
+
+    kwargs get passed on to Elasticsearch
+
+    .. seealso:: https://elasticsearch-py.readthedocs.io/en/master/api.html#elasticsearch
+
+    :param alias: The alias to use for referencing the created connection. Will
+                  create a "default" connection unless this param has a value
+                  other than `default`
+    :param hosts: List of host(s) (e.g. 'http://foo.com:9200') to connect to
     """
-    # TODO invoke this during request bootstrapping
-    # TODO settings munging if/as needed
-    # TODO AWS auth
-    # TODO certs
-    connections.create_connection(alias, hosts=hosts, **kwargs)
+    connections.create_connection(alias,
```

Comment:
These args assume that these things are true:

* no AWS auth needed anymore
* no need for `certifi.where` to locate certs anymore